### DISCUSSION TITLE
Default Project Path

### DISCUSSION THREAD

**Discussion by TechCowboy** [Redot-Engine/redot-proposals#61] [Ideas]
One "feature" that has always bugged me:
You can't change the default project path until you create a project.
I feel this should really be in the Project List under settings.

**Comment by Arctis-Fireblight** [comment]
I feel like this should be a very basic feature, and am shocked that it doesnt already exist, and had to double check the editor to make sure...

I am greenlighting this for inclusion for Redot 26.2
